testkit-specs2-5/src/main/scala/com/kevel/apso/AdditionalIsEmpty.scala

```diff
@@ -0,0 +1,22 @@
+package com.kevel.apso
+
+import org.specs2.collection.IsEmpty
+
+trait AdditionalIsEmpty {
```

Comment:
I think we should be good with a single instance for `IterableOnce` (and also support more types along with it), but I wonder if this shouldn't be allowed upstream instead.

Reply:
This should not be required with [specs2 5.6.4](https://github.com/etorreborre/specs2/pull/1345).